tools/wtdbg/wtdbg.xml

```diff
@@ -0,0 +1,195 @@
+        $rescue_low_cov_edges
+
+        &&
+        
+        wtdbg-cns 
+        -t \${GALAXY_SLOTS:-4} 
+        -o dbg.ctg.lay.fa
+        -i dbg.ctg.lay
+        -j $cns.j
+        -k $cns.k
+        -Z $cns.Z
+        -W $cns.W
+        -H $cns.H
+        -L $cns.L
+        -c $cns.c
+        -M $cns.M
+        -X $cns.X
+        -I $cns.I
+        -D $cns.D
+        -E $cns.E
+        -m $cns.m
+        -S $cns.S
+        $v
+        
+        
+    ]]></command>
+    <inputs>
+        <param type="data" argument="-i" format="fasta,fasta.gz" label="Long reads sequences file"/>
+        <param type="data" argument="-I" format="fasta,fasta.gz" optional="True" label="Error-free sequences file"/>
+        <param type="text" argument="-o" value="dbg" label="Prefix of output files"/>
+
+        <param argument="k" type="integer" value="0" min="0" max="25" label="Kmer fsize" />
+        <param argument="p" type="integer" value="21" min="0" max="25" label="Kmer psize" />
+        <param argument="K" type="float" value="1000" min="0" max="65535" label="Filter high frequency kmers" />
+        <param argument="E" type="integer" value="2" label="Min kmer frequency" />
+        <param argument="F" type="boolean" truevalue="-F" falsevalue="" checked="False" label="Filter low frequency kmers by a 4G-bytes array" />
+        <param argument="S" type="integer" value="4" label="Subsampling kmers, 1/S kmers are indexed" />
+        <param argument="X" type="integer" value="4" label="Max number of bin(256bp) in one gap" />
+        <param argument="Y" type="integer" value="4" label="Max number of bin(256bp) in one deviation" />
+        <param argument="x" type="integer" value="-7" label="penalty for BIN gap" />
+        <param argument="y" type="integer" value="-21" label="penalty for BIN deviation" />
+        <param argument="l" type="float" value="2048" label="Min length of alignment" />
+        <param argument="m" type="float" value="200" label="Min matched" />
+        <param argument="s" type="float" value="0.2" label="Max length variation of two aligned fragments" />
+        <param argument="v" type="boolean" truevalue="-v" falsevalue="" checked="False" label="Verbose" />
+
+        <param argument="--tidy-reads" name="tidy_reads" type="integer" value="0" label="Filter reads less than tidy-reads" />
+        <param argument="--edge-min" name="edge_min" type="integer" value="3" label="The minimal depth of a valid edge set to" />
+        <param argument="--rescue-low-cov-edges" name="rescue_low_cov_edges" type="boolean" truevalue="--rescue-low-cov-edges" 
+          falsevalue="" label="Try to rescue low coverage edges" />
+
+        <section name="cns" title="Consensus options">
+            <!-- optional inputs -->
+            <!-- <param argument="-i" type="data" format="utg.cns" label="Input file(s) *.utg.cns" /> -->
+
+            <param argument="-j" type="integer" value="1000" label="Expected length of node" />
```

Comment:
can we add min/max to anyone of those?

Reply:
Those who had limits mentioned in help are already added.